File: get_dataset.py
import requests
from bs4 import BeautifulSoup

response = requests.get("https://www.dailymail.co.uk/news/article-2523930/Jesus-famous-person-history-according-software-algorithm.html")
soup = BeautifulSoup(response.content, 'html.parser')

div_content = soup.find('div', class_='artSplitter art-ins news')
famous_people = []

if div_content:
  for element in div_content.select("p b"):
    famous_people.append(element.text.strip())

famous_people.reverse()

with open("data.csv", "w") as file:
  for person in famous_people:
    file.write(f'{person},\n')

File: get_dataset/get_dataset.js
import { writeFileSync } from 'node:fs'
import puppeteer from 'puppeteer'

const browser = await puppeteer.launch()
const page = await browser.newPage()
await page.goto("https://www.dailymail.co.uk/news/article-2523930/Jesus-famous-person-history-according-software-algorithm.html", { waitUntil: 'networkidle2' })

const famous_people = (await page.$$eval('p b', element => element.map(name => name.textContent)))

await browser.close()

writeFileSync('data.txt', famous_people.join('\n'), 'utf-8')